bands/templates/emails/band_merch_reminder.txt

```diff
@@ -1,6 +1,6 @@
 {{ band.group.name }},
 
-{{ c.EVENT_NAME }} encourages performers to sell their merchandise throughout the event.  The best way to do this is to participate in "Rock Island", where our volunteers sell your merchandise at a booth which is staffed throughout the event.  Bands can opt to instead sell from their own dedicated table in the Marketplace if they are will to staff for at least 8 hours per day.  We need you to tell us which option you prefer by filling out the form at {{ c.URL_BASE }}/bands/rock_island?id={{ band.id }}
+{{ c.EVENT_NAME }} encourages performers to sell their merchandise throughout the event.  The best way to do this is to participate in "Rock Island", where our volunteers sell your merchandise at a booth which is staffed throughout the event.  Bands can opt to instead sell from their own dedicated table in the Marketplace if they are will to staff for at least 8 hours per day.  We need you to tell us which option you prefer by filling out the form at {{ c.URL_BASE }}/bands/merch?band_id={{ band.id }}
```

Comment:
The base url of `rock_island` ALSO didn't exist anymore. Looks like these emails haven't been updated in a while.